config/database.yml

```diff
@@ -29,10 +29,10 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: targetmvdapi
+  username: keiny
 
   # The password associated with the postgres role (username).
-  #password:
+  password: t4rg3tmvd4p1
```

Comment:
You should not commit this configuration file.
Your password is exposed!

You can add it to git ignore file 😁

Reply:
Try removing this file from the PR.